src/movie/bill.js

```diff
@@ -8,7 +8,39 @@ module.exports = class Bill {
      * @param loge      true if seating category is 'loge')
      * @param threeD    true if the movie's shown in 3D
      */
+
+    ticketPrices = [];
+    extraPrice = 0.0;
+    specialMovieDay = false;
+
     startPurchase(runtime, dayOfWeek, loge, threeD) {
```

Comment:
Deze methode begint wat lang te worden, denk is na hoe je deze zou kunnen opsplitsen.